### ISSUE TITLE
Tag Mismatch Warnings

### ISSUE THREAD

**Issue by denisbranisteanu** [TheManyula/progress3D#2]
Hello, if i use the latest version of zeex compiller (3.10.10) with this include i get only errors:

```
\pawno\include\progress3D.inc(40) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(41) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(66) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(66) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(67) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(67) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(71) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(71) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(72) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(72) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(84) : warning 213: tag mismatch: expected tag "Bar3D", but found none ("_")
\pawno\include\progress3D.inc(86) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(86) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(88) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(89) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(92) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(93) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(97) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(98) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(99) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(100) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(101) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(133) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(134) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(135) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(136) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(138) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(139) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(141) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(142) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(143) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(144) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(163) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(167) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(167) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(167) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(167) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(168) : warning 213: tag mismatch: expected tag "Text3D", but found none ("_")
\pawno\include\progress3D.inc(226) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(226) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(227) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(228) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(231) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(232) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(236) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(237) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(238) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(239) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(240) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(271) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(272) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(273) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(274) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(276) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(277) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(279) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(280) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(281) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(282) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(301) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(301) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(301) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(301) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
\pawno\include\progress3D.inc(302) : warning 213: tag mismatch: expected tag "PlayerText3D", but found none ("_")
```

**Comment by denisbranisteanu** [comment]
i fixed the errors by putting manually the value for max's

```
#define MAX_GLOBAL_3D_BARS    1024
#define MAX_PLAYER_3D_BARS    1024
```

**Comment by TheManyula** [comment]
Thanks for reporting! I've just fixed the issue and released [v2.3.2](https://github.com/TheManyula/progress3D/releases/tag/v2.3.2).
